### ISSUE TITLE
Addons: activating a version doesn't purge the cache

### ISSUE THREAD

**Issue by humitos** [readthedocs/readthedocs.org#11291]
When the use activates a new version, the addons API response doesn't immediately reflect this change. This is because the API response is cached. However, triggering a purge on that project won't immediately work since the version won't be `built=True` at that point yet.

So, we need to figure it out how to purge the cache on "new active versions, but _only once_ they have finished successfully". How we can detect this?

**Comment by humitos** [comment]
We can't use `build_complete` signal because at that time we don't know yet if the build was successful or not. I opened https://github.com/readthedocs/readthedocs.org/pull/11489 to handle this case.

**Comment by ericholscher** [comment]
Did we solve this, or should we look at re-enabling https://github.com/readthedocs/readthedocs.org/pull/11489 ?

**Comment by humitos** [comment]
Yeah, we should probably revive that PR and merge it. We didn't migrate addons to multiple API endpoints yet and we are not going to do it soon, I guess. If you want you can re-open it and jump in that conversation. I see it pretty clear but I got de-motivated by the replies.
